### PR TITLE
fix(theme-classic): fixed wrong cursor on dropdown menu in navbar, when window is small

### PR DESCRIPTION
**Pull request by Shreesh09**
## Pre-flight checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [ ] **If this is a code change**: I have written unit tests and/or added dogfooding pages to fully verify the new behavior.
- [ ] **If this is a new API or substantial change**: the PR has an accompanying issue (closes #0000) and the maintainers have approved on my working plan.

## Motivation
Fixed the issue where wrong cursor was displaying when hovering over any dropdown menu present in the navbar, when the screen size is small

## Test Plan

Incorrect cursor:
without-fix

Correct cursor:
with-fix

### Test links

Deploy preview: https://deploy-preview-_____--docusaurus-2.netlify.app/

## Related issues/PRs
#8485

### CODE REVIEW

**Comment by Josh-Cena** [comment]
Please use CSS instead. This link is not a link, so it should not have an `href`.

**Comment by Shreesh09** [comment]
> Please use CSS instead. This link is not a link, so it should not have an `href`.

I am simply replicating how the dropdown menu currently behaves in the navbar when the screen size is normal, is that incorrect aswell?

dropdown

**Comment by Shreesh09** [comment]
Hello @Josh-Cena, I have implemented the changes. Could you please take a moment to review them? Thank you.

File: packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem/index.tsx [hunk 1]
```diff
@@ -137,7 +138,7 @@ function DropdownNavbarItemMobile({
 
   return (
     <li
-      className={clsx('menu__list-item', {
+      className={clsx(styles.dropdownNavbarItemMobile, 'menu__list-item', {
```
Comment (Josh-Cena) [inline packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem/index.tsx:141]: Why is it called `dropdownNavbarItemMobile`? It's used on desktop, right?

**Review by Josh-Cena — APPROVED** [review]
You would need to format your code. You can copy this file into https://prettier.io/playground/ if you don't have a local clone. Otherwise, this should be good.

File: packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem/index.tsx (hunk 1 above)
Comment (Josh-Cena) [reply packages/docusaurus-theme-classic/src/theme/NavbarItem/DropdownNavbarItem/index.tsx:141]: Never mind, this is only for mobile :)

**Comment by Shreesh09** [comment]
I believe the code is properly formatted because after copy pasting from prettier and then commiting the code remains unchanged

**Comment by Josh-Cena** [comment]
I see. You are right.

**Review by Josh-Cena — APPROVED** [review]
LGTM, thanks :)

**Review by Josh-Cena — CHANGES_REQUESTED** [review]
Actually, instead of applying this class to the `<li>`, we should apply it to the inner `<NavbarNavLink>`, because that's the one that should theoretically be a pointer, should there be an `href`. Do you want to do this change? I will do it in the upcoming days otherwise.

**Comment by Shreesh09** [comment]
Sure, I will make the changes.

**Review by Josh-Cena — APPROVED** [review]
Thanks for the timely responses!

**Pull request merged**